docs/DeveloperGuide.md

```diff
@@ -1040,13 +1022,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1.  User requests to delete several clients at once.
 
-2.  ClientBook removes the client contacts from the list.
+2.  ClientBook removes the client contacts from the address book.
```

Comment:
The sentence could use the wording in delete use case: ClientBook deletes the clients.